src/main/java/net/mcreator/ui/minecraft/villagers/ProfessionListField.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.villagers;
+
+import net.mcreator.element.parts.ProfessionEntry;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.JItemListField;
+import net.mcreator.ui.dialogs.DataListSelectorDialog;
+import net.mcreator.ui.init.L10N;
+
+import java.util.List;
+
+public class ProfessionListField extends JItemListField<ProfessionEntry> {
+
+	public ProfessionListField(MCreator mcreator) {
+		super(mcreator);
```

Comment:
It would be interesting to allow `include/exclude` options. It's too different from the current code and too hard to implement.

Once this will be implemented, it should be good an approval for me.